### PR TITLE
feat: add wallet

### PR DESCRIPTION
**Pull request by vladopajic**

### CODE REVIEW

File: pkg/funder/funder.go [hunk 1]
```diff
@@ -6,21 +6,190 @@ package funder
...
+	respC <- func() error {
+		cid := node.WalletInfo.ChainID
+
+		token, err := wallet.NativeCoinForChain(cid)
+		if err != nil {
+			return fmt.Errorf("%w: %s", ErrFailedFudningNodeWithNativeCoin, err)
+		}
+
+		topUpAmount := CalcTopUpAmount(minAmounts.NativeCoin, node.WalletInfo.NativeCoin, token.Decimals)
+		if topUpAmount.Cmp(big.NewInt(0)) <= 0 {
```
Comment (umerm-work) [inline pkg/funder/funder.go:135]: Not sure if I understand this. Does this mean that we are checking whether the current amount is below or above min required amount?

Comment (umerm-work) [reply pkg/funder/funder.go:135]: My understanding is that we just top up regardless of how much bzz or eth is in per address.

Comment (vladopajic) [reply pkg/funder/funder.go:135]: Yes, we are check current balance and only top up to the min amount (specified when starting).

Comment (vladopajic) [reply pkg/funder/funder.go:135]: > My understanding is that we just top up regardless of how much bzz or eth is in per address.

Why should node be funded if it already has enough bzz and eth?

**Pull request merged**